build.gradle

```diff
@@ -70,7 +70,3 @@ shadowJar {
 }
 
 defaultTasks 'clean', 'test'
-
-run {
-    enableAssertions = true
-}
```

Comment:
Why is this removed? Did you forget to pull before starting on you edits?

Reply:
Added it back